doc/memos/rdm-draft-alamos-ieee802154-radio-hal.md

```diff
@@ -0,0 +1,967 @@
+- RDM: rdm-draft-alamos-ieee802154-radio-hal
```

Comment:
According to RDM 0, this document should now be renamed `rdm0002.md` and this line changed to

```suggestion
- RDM: 2
```